app/javascript/components/ArtistRequests.jsx

```diff
@@ -0,0 +1,226 @@
+              item.work = data;
+              this.setState({ inbox: this.state.inbox.concat(item), display: this.state.display.concat(i) });
+            });
+          });
+        },
+        error => {
+          console.error(error);
+        }
+      );
+    });
+  }
+
+  getAttr = (request) => {
+    console.log(request);
+    let attr = {
+      "Buyer": request.buyer.name,
+      "Artist": request.artist.name,
+      "Placed": new Date(request.request.updated_at).toLocaleDateString(),
+      "Request Type": request.request.types
+    };
+
+    if (!request.open && request.receipt) {
+      if (request.receipt.transaction_type === "rental") {
+        attr["Start Date"] = request.receipt.start_date;
+        attr["End Date"] = request.receipt.end_date;
+      }
+      attr["Price"] = request.receipt.price;
+      attr["Purchase Date"] = request.receipt.purchase_date;
+    }
+
+    return (
+      <div className="attr">
+        <div className="key">
+          {
+            Object.keys(attr).map((obj, i) => {
+              return <h5 key={i} className="attr-item">{obj}</h5>
+            })
+          }
+        </div>
+        <div className="value">
+          {
+            Object.keys(attr).map((obj, i) => {
+              return <h6 key={i} className="attr-item">{attr[obj]}</h6>
+            })
+          }
+        </div>
+      </div>
+    );
+  }
+
+  RequestListItem = (props) => {
+    const request = props.request;
+    const id = request.request.id;
+    const buyer = request.buyer;
+    const artist = request.artist;
+    const title = request.work.title;
+    const thumbnail_url = request.work.thumbnail ? request.work.thumbnail : "https://cdn0.iconfinder.com/data/icons/typicons-2/24/image-128.png";
+    const open = request.request.open;
+    const created_timestamps = new Date(request.request.created_at).toLocaleDateString();
+    const closed_timestamps = new Date(request.request.updated_at).toLocaleDateString();
+
+    return (
+      <div className="content-row">
+        <div className="request-content">
+          <div>
+            <img src={thumbnail_url} className="w4"/>
+          </div>
+          <div className = "items">
+            {this.getAttr(request)}
+          </div>
+        </div>
+        <div className="request-buttons w5">
+          {
+            request.request.open ? (
+              <div className="w-100">
+                <StyledModal title="MARK AS COMPLETE">
+                  <CreateTransaction
+                    artist={this.props.artist}
+                    request_id={id}
+                  />
+                </StyledModal>
+                <button type="button" className="button-secondary b--charcoal w-100 mt2" value = {id} onClick = {()=>this.closeRequest(id)}>CLOSE REQUEST</button>
+              </div>
+            ) : (
+              <div className = "closed-request-button pa4 w-100">
+                <p> You closed this request on {closed_timestamps} </p>
```

Comment:
There should probably be a more specific message for completed requests since 'closed' is the equivalent of 'rejected' - it's kind of confusing that this pops up after an artist completes a request by recording the receipt

Reply:
Thank you! Renamed to "reviewed"